equinix-openapi-metal/src/main/java/com/equinix/openapi/metal/v1/model/AttributeData.java

```diff
@@ -240,17 +240,18 @@ private String toIndentedString(Object o) {
   }
 
  /**
-  * Validates the JSON Object and throws an exception if issues found
+  * Validates the JSON Element and throws an exception if issues found
   *
-  * @param jsonObj JSON Object
-  * @throws IOException if the JSON Object is invalid with respect to AttributeData
+  * @param jsonElement JSON Element
+  * @throws IOException if the JSON Element is invalid with respect to AttributeData
   */
-  public static void validateJsonObject(JsonObject jsonObj) throws IOException {
-      if (jsonObj == null) {
-        if (!AttributeData.openapiRequiredFields.isEmpty()) { // has required fields but JSON object is null
+  public static void validateJsonElement(JsonElement jsonElement) throws IOException {
```

Comment:
This is another repeated change here, `validateJsonObject` changing to `validateJsonElement`.  Internally it seems to be non-impacting, but since it's a public method it's technically a breaking change.